renovate.json

```diff
@@ -3,10 +3,44 @@
   "extends": [
     "config:recommended"
   ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["^\\.github/workflows/build_publish_builder_image\\.yaml$"],
+      "matchStrings": [
+        "go-version:\\s*\\[\\s*(?<currentValue1>1\\.22\\.\\d+),\\s*(?<currentValue2>1\\.23\\.\\d+)\\s*\\]"
+      ],
+      "datasourceTemplate": "golang-version",
+      "depNameTemplate": "golang-1.22",
+      "currentValueTemplate": "{{{currentValue1}}}",
+      "extractVersionTemplate": "(?<version>1\\.22\\.\\d+)"
+    },
+    {
+      "fileMatch": ["^\\.github/workflows/build_publish_builder_image\\.yaml$"],
+      "matchStrings": [
+        "go-version:\\s*\\[\\s*(?<currentValue1>1\\.22\\.\\d+),\\s*(?<currentValue2>1\\.23\\.\\d+)\\s*\\]"
+      ],
+      "datasourceTemplate": "golang-version",
+      "depNameTemplate": "golang-1.23",
+      "currentValueTemplate": "{{{currentValue2}}}",
+      "extractVersionTemplate": "(?<version>1\\.23\\.\\d+)"
+    }
```

Comment:
Shouldn't this also have `"customType": "regex"`?

Reply:
yep missed adding here :(, added it now